src/Chapter07/Listing07.31.SwitchWithPatternMatching.cs

```diff
@@ -27,7 +27,7 @@ input switch
             compositeFormatString, date.Year, date.Month, date.Day) : null;
     #endregion INCLUDE
 }
-file static class DateDeconstructors
+static file class DateDeconstructors
```

Comment:
This one surprises me since the docs say that `file` should come before `static`
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0036#csharp_preferred_modifier_order

Reply:
I agree, I opened an issue and pr on Roslyn to see if this needs an update, going to wait to merge this until I hear back on that, as I might need to dial down the warning to a message if they have a mistake until it's patched. https://github.com/dotnet/roslyn/issues/70702

Reply:
Sounds good to me. I think for now I would suggest switching it back as I suspect that is the most likely default.

Reply:
Reverted!

Reply:
Gonna merge this for now. If something changes I'll revert